<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_dialog</class>
 <widget class="QDialog" name="edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="rever_button">
     <property name="text">
      <string>Revert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="submit_button">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancel_button">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Product</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="brand_label">
        <property name="text">
         <string>Brand:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="manufactory_label">
        <property name="text">
         <string>Manufactory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="brand_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="address_label">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="year_label">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="detail_label">
        <property name="text">
         <string>Detail(Separated by ';'):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="manufactory_edit"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="year_edit">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="address_edit"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="detail_edit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
